README.md

```diff
@@ -68,9 +68,7 @@ GraphQL Client:
 
 ## Example <a name="examples"></a>
 
-Make sure to check out more examples in the Console Test App in the package. Down here is an example call for connecting with a Web API service through Refit:
-
-The Console App is locating here:[Console App](TestConsoleApp)
+Make sure to check out more examples in the [Test Console App](TestConsoleApp) in the package. Down here is an example call for connecting with a Web API service through Refit:
```

Comment:
Now that #63 is merged could you add the correct link to the console app here?

Reply:
Yes I will